src/zenml/model/model_config.py

```diff
@@ -151,18 +118,25 @@ def _validate_config_in_runtime(self) -> None:
         """
         try:
             model_version = self._get_model_version()
-            if self.create_new_model_version:
+            if self.version is None or self.version == RUNNING_MODEL_VERSION:
+                self.version = RUNNING_MODEL_VERSION
                 for run_name, run in model_version.pipeline_runs.items():
                     if run.status == ExecutionStatus.RUNNING:
                         raise RuntimeError(
-                            f"New model version was requested, but pipeline run `{run_name}` "
-                            f"is still running with version `{model_version.name}`."
+                            f"New unnamed model version was requested, "
+                            f"but pipeline run `{run_name}` have not finished yet. "
+                            "This run also operates with unnamed model version - "
+                            "new run will be stopped to prevent unexpected behavior."
```

Comment:
I don't fully get the error message - would be better if you explain what the user is supposed to do now? Rename his version? Or wait for this other run to complete?

Reply:
reworked